code/game/objects/items/weapons/shards.dm

```diff
@@ -87,6 +87,7 @@
 				H.Weaken(3)
 				if(affecting.take_damage(5, 0))
 					H.UpdateDamageIcon()
+					affecting.embed(src)
```

Comment:
Is there a way to remove stepping on anything when there is no gravity? Is that not the case already, I don't know